package.json

```diff
@@ -17,5 +17,8 @@
   },
   "scripts": {
     "install": "prades install"
+  },
+  "dependencies": {
+    "@sb/prades": "^0.4.0"
```

Comment:
@TrustNik @RemiArnaud - Vadim and I had a skype call previously that prompted this PR. At the time, Vadim asserted that local installs were the norm. Since then I've researched and find that it's quite rare for packages to have a `dependencies` entry for `npm` itself.

What I do see often is an `engines` object with `node` and `npm`. It would seem to me that tools like `yarn` and `prades` also belong in the `engines` object. There are examples of this in docs (https://yarnpkg.com/lang/en/docs/package-json/#toc-engines) and tools (https://www.npmjs.com/package/check-engine). It appears that items in the `engines` object are expected to exist on the client which is analogous to being pre-installed and i.e. under configuration management.

Also npm best practices says not to use `install` scripts. See https://docs.npmjs.com/misc/scripts#best-practices , something this approach is doing.

Also @dgaya  endorses local dependency for `prades` as implemented in this and similar PR. See https://github.com/fl4re/prades/blob/master/Readme.md . 

There have been plenty of issues regarding "npm dependency hell" and attempts to solve them like `peerDependencies`. So I'm not convinced that what's been discussed previously (between any of us) and decided upon as the best approach has been consistent and valid.